src/widgets/refdata/list-ui.vue

```diff
@@ -5,7 +5,7 @@
     <b-form-group id="list-ui-form-group" class="list-ui-form-group" :class="isRequired ? 'required' : ''" :label-cols="uiLabel ? labelCols : 1" :description="uiDescription" :label="uiLabel">
       <b-input-group>
         <div v-if="type === 'multiselect'" class="flex-grow-1">
-          <Multiselect v-model="model" @input="setValue" :options="items" v-if="items" :customLabel="customLabel"
```

Comment:
This gives an error when you open the files tab and have a file selected in the dataset. The list-ui component (used by FileItem.vue) does not have the `property` prop.

Reply:
:/

Reply:
removed the change